livekit_rtc.proto

```diff
@@ -217,3 +219,9 @@ message SubscriptionPermissionUpdate {
   string track_sid = 2;
   bool allowed = 3;
 }
+
+message SyncState {
+  SessionDescription anwser = 1;
```

Comment:
Can we name this `subscriber_answer` to indicate that it corresponds to subscriber peer connection?

Reply:
In our SignalRequest, answer is only used for subscribe peer connection, so use answer for consistent